recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <span class="d-d-flex d-fw-wrap">
+    <span
+      v-for="reaction in reactions"
+      :key="reaction.unicodeOutput"
+      :reaction="reaction"
+      class="d-mr4 d-mb4"
+    >
+      <dt-tooltip
+        class="d-d-inline-block"
+        content-class="d-wmx464"
+        @shown="(shown) => emojiHovered(reaction, shown)"
+      >
+        <dt-emoji-text-wrapper size="200">
+          {{ reaction.tooltip }}
+        </dt-emoji-text-wrapper>
+        <template #anchor="{ attrs }">
+          <dt-button
+            importance="clear"
+            size="sm"
+            data-qa="feed-item-reaction-button"
+            :class="[
+              'd-h24 d-bar16 d-py0 d-fs-200 d-mb0 d-t',
+              !!reaction.isSelected
+                ? `d-fc-purple-400 d-bgc-purple-400 d-bgo10 d-bc-purple-400 d-bco50 ' +
+                  'h:d-bgc-purple-400 h:d-bco25 a:d-bgc-purple-400 a:d-bco25`
+                : 'd-fc-black-600 d-bgc-black-100 h:d-bgc-white h:d-bc-black-600 a:d-bgc-white a:d-bc-black-600',
+            ]"
+            :aria-label="reaction.ariaLabel"
+            :attrs="attrs"
+            @click="emojiClicked(reaction)"
+          >
```

Comment:
hmmm color contrast problem when it's selected.. might want to bring it up with design.

![Screenshot 2023-05-09 at 4 05 00 PM](https://github.com/dialpad/dialtone-vue/assets/64808812/44d6a9d7-c4dd-4fec-9a5b-5fc3da3f8114)

Reply:
Talked to Bradley, will be bumping this to fc-purple-500